docs/debugger/how-to-install-a-visualizer.md

```diff
@@ -24,7 +24,33 @@ After you have created a visualizer, you must install the visualizer so that it
 > [!NOTE]
 > In UWP apps, only the standard text, HTML, XML, and JSON visualizers are supported. Custom (user-created) visualizers are not supported.
 
-### To install a visualizer
+### To install a visualizer for Visual Studio 2019
+  
+1. Locate the DLL that contains the visualizer you have built.
+
+2. Copy the [Debugger Side](create-custom-visualizers-of-data.md#to-create-the-debugger-side) DLL to either of the following locations:
+
+    - *VisualStudioInstallPath* `\Common7\Packages\Debugger\Visualizers`
+
+    - `My Documents\` *VisualStudioVersion* `\Visualizers`
+    
+3. Copy the [Debuggee Side](create-custom-visualizers-of-data.md#to-create-the-debuggee-side) DLL to either of the following locations:
+
+    - *VisualStudioInstallPath* `\Common7\Packages\Debugger\Visualizers\` *Framework*
+
+    - `My Documents\` *VisualStudioVersion* `\Visualizers\` *Framework*
+
+    Where *Framework* is either:
+    - `net2.0` for debuggees running the `.NET Framework` runtime.
+    - `netstandard2.0` for debuggees using a runtime supported in `netstandard 2.0` (supports .NET Framework v4.6.1 and greater and .NET Core 2.0, 2.1 and 2.2).
```

Comment:
You don't want to list all the .NET Core versions here - it will support all others in the future. Suggested text `(supports .NET Framework v4.6.1+ and .NET Core 2.0+)`